src/luascript.cpp

```diff
@@ -13957,7 +13929,31 @@ int LuaScriptInterface::luaSpellCreate(lua_State* L)
 			setMetatable(L, -1, "Spell");
 			return 1;
 		}
+
+		std::string tmp = asLowerCaseString(getString(L, 2));
```

Comment:
`arg` can be reused here: `asLowerCaseString(arg)`

I would also prefer a different name for the variable e.g. `spellType`